web/src/lib/components/shared-components/side-bar/side-bar-link.svelte

```diff
@@ -51,6 +55,24 @@
     <span class="text-sm font-medium">{title}</span>
   </div>
 
+  {#if hasDropdown}
+    <span class="hidden md:block">
+      <button
+        type="button"
+        class="relative flex cursor-default select-none justify-center hover:cursor-pointer hover:bg-immich-gray hover:text-immich-primary dark:text-immich-dark-fg dark:hover:bg-immich-dark-gray dark:hover:text-immich-dark-primary rounded"
+        onclick={() => (dropdownOpen = !dropdownOpen)}
+      >
+        <Icon
```

Comment:
What do you think about putting this icon on the left side? It feels strange to use by having it on the right side and the tooltip blocks it when you hover over the `i` icon.

Reply:
What about this?
![image](https://github.com/user-attachments/assets/3aaf1e58-4aa5-462b-838d-c61f77ae99ac)